invenio_communities/assets/semantic-ui/js/invenio_communities/members/members/manager_view/ManagerMembersResultItem.js

```diff
@@ -77,20 +73,33 @@ export class ManagerMemberViewResultItem extends Component {
     // TODO: membersApi.edit(result); + NOTIFICATIONS
   };
 
+  openLeaveOrRemoveModal = (isRemoving = true) => {
+    const { openModal } = this.context;
+    const { result, membersApiClient } = this.props;
+
+    const {
+      member: { type, id },
+    } = result;
+    const payload = { members: [{ type, id }] };
```

Comment:
I think it would be better to move this serialization to the API, since the component does not need to know about the payload structure